jonprl-mode.el

```diff
@@ -417,6 +433,7 @@ Invokes `jonprl-mode-hook'."
   (setq-local comment-start "|||")
   (jonprl-update-available-tactics)
   (setq-local font-lock-defaults (jonprl-font-lock-defaults))
+  (setq-local prettify-symbols-alist jonprl-pretty-symbols)
```

Comment:
A hook approach could work. However, the user should be able to just modify the alist in their own configuration too. Do you think the hook approach would be superior to that? I haven't though about it much but am fine changing it.